components/org.wso2.carbon.business.rules.core/pom.xml

```diff
@@ -79,6 +79,13 @@
             <artifactId>slf4j-log4j12</artifactId>
         </dependency>
 
+        <!-- GSON dependencies-->
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.2</version>
```

Comment:
The version should go to the parent pom. Here, it should be only group id and artifact id

Reply:
Changed